components/elm/src/biogeophys/SoilHydrologyType.F90

```diff
@@ -542,18 +547,36 @@ subroutine InitCold(this, bounds)
        fdrain(:) = 2.5_r8
     end if
     call ncd_pio_closefile(ncid)
+    
+    call getfil (fsurdat, locfn, 0)
+    call ncd_pio_openfile (ncid, locfn, 0)
+    call ncd_io(ncid=ncid, varname='fover', flag='read', data=this%fover, dim1name=grlnd, readvar=readvar)
+    if (.not. readvar) then
+      this%fover(:) = 0.5_r8
+    end if
+    call ncd_pio_closefile(ncid)
+    
+    call getfil (fsurdat, locfn, 0)
+    call ncd_pio_openfile (ncid, locfn, 0)
+    call ncd_io(ncid=ncid, varname='pc', flag='read', data=this%pc, dim1name=grlnd, readvar=readvar)
+    if (.not. readvar) then
+      this%pc(:) = 0.4
```

Comment:
Change `0.4` to `0.4_r8`

Reply:
Using `0.4_r8` will result in non BFB to the master branch. That is because `pc` is set to 0.4 in the master (Please see next reply).

Reply:
In that case, leave it as is. Thanks.